exercise_06_b/pipeline.txt

```diff
@@ -0,0 +1,23 @@
+pipeline {
+    agent {label 'jenkins-slave-archita'}
+    
+    parameters {
+        string(defaultValue: "https://github.com/infuse-training-jan-2023/practice_archita.git", description: 'Github repository url', name: 'github_url')
```

Comment:
this is not the correct way of adding parameters, the url is visible and you can find another way